.travis.yml

```diff
@@ -9,23 +9,29 @@ cache:
 env:
   global:
     - HYPEROPT_FMIN_SEED=3
-    - SPARK_VERSION=2.4.4
+    - SPARK_VERSION=3.0.1
```

Comment:
With official Spark 3.0.x release, we don't need to manually download the Spark distribution jar. We just need to define the Spark version like numpy. We can also clean up download_spark_dependencies.sh.